host.go

```diff
@@ -361,20 +361,20 @@ func (h *gpbftHost) collectChain(base ec.TipSet, head ec.TipSet) ([]ec.TipSet, e
 	res := make([]ec.TipSet, 0, 2*gpbft.ChainMaxLen)
 	res = append(res, head)
 
-	for !bytes.Equal(head.Key(), base.Key()) {
-		if head.Epoch() < base.Epoch() {
-			// we reorged away from base
-			// scream and panic??
-			// TODO make sure this is correct, re-boostrap/manifest swap code has to be able to
-			// catch it
-			panic("reorg-ed away from base, dunno what to do, reboostrap is the answer")
+	current := head
+	for !bytes.Equal(current.Key(), base.Key()) {
+		if current.Epoch() < base.Epoch() {
+			metrics.headDiverged.Add(h.runningCtx, 1)
+			log.Errorw("reorg-ed away from base, reboostrap is the answer",
```

Comment:
This is going to _majorly_ annoy users.

Reply:
It will happen only every minute or so, but agreed, I will downgrade to Warn and tone it down